### PR TITLE
Added some documentation to particles

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Just added a few missing descriptions. If someone can make suggestions for capture_aabb, fixed_fps and fract_delta I'll add those in to complete this.

### CODE REVIEW

File: doc/classes/Particles.xml [hunk 1]
```diff
@@ -33,14 +34,19 @@
 			Particle draw order. Uses [code]DRAW_ORDER_*[/code] values. Default value: [code]DRAW_ORDER_INDEX[/code].
 		</member>
 		<member name="draw_pass_1" type="Mesh" setter="set_draw_pass_mesh" getter="get_draw_pass_mesh">
```
Comment (mhilbrunner) [inline doc/classes/Particles.xml]: Shorten these to "[Mesh] that is drawn for the X draw pass."

File: doc/classes/Particles.xml [hunk 2]
```diff
@@ -22,6 +22,7 @@
 			<return type="void">
 			</return>
 			<description>
+				Restarts the particle emmission clearing existing particles.
```
Comment (mhilbrunner) [inline doc/classes/Particles.xml]: Add comma: Restarts the particle emission, clearing existing particles.

File: doc/classes/Particles.xml [hunk 3]
```diff
@@ -62,6 +68,7 @@
 			If [code]true[/code] only [code]amount[/code] particles will be emitted. Default value: [code]false[/code].
 		</member>
 		<member name="preprocess" type="float" setter="set_pre_process_time" getter="get_pre_process_time">
+			Amount of time to preprocess the particles before animation starts. Let's you start the animation some time after particles have started emitting.
```
Comment (mhilbrunner) [inline doc/classes/Particles.xml]: Lets

**Review by mhilbrunner — COMMENTED** [review]
Thanks :) Comments:

**Comment by mhilbrunner** [comment]
@BastiaanOlij Poke :)

**Comment by BastiaanOlij** [comment]
totally missed your feedback @mhilbrunner , thanks for the poke, I'll get on this later in the morning :)

**Comment by ronroniv** [comment]
I think this can be written together with Particles2D document, as they have some common properties. I see [Particles2D doc](http://docs.godotengine.org/en/latest/classes/class_particles.html) doesn't have description on `restart()` and already have one for `preprocess`.

**Comment by mhilbrunner** [comment]
@BastiaanOlij Merging this one, you can create another PR for the 2D particles doc using some of the content here if you want :)

**Comment by akien-mga** [comment]
Cherry-picked into 3.0.4.

**Pull request merged**